can-component.js

```diff
@@ -253,12 +269,63 @@ var Component = Construct.extend(
 					options.tags = {};
 				}
 
+				options.parentRenderer = this.parentRenderer;
+				options.parentViewModel = this.viewModel;
+				options.parentTag = this.parentTag;
+
+				options.tags.super = function superHookup(el, superTagData) {
```

Comment:
Yeah, I totally agree! The code between `content` and `super` is basically identical, and I presume that `slot` will be similar. I'll reach out.